tests/test_slackrequest.py

```diff
@@ -111,5 +112,5 @@ def test_post_attachements(mocker):
 
     assert requests.post.call_count == 1
     assert 'https://slack.com/api/chat.postMessage' == args[0]
-    assert {'attachments': [{'title': 'hello'}]} == kwargs['data']
-    assert kwargs['files'] is None
+    assert {'attachments': '[{"title": "hello"}]'} == kwargs['data']
+    assert isinstance(kwargs["data"]["attachments"], str)
```

Comment:
Minor nit: I think L116 in the test is t actually testing anything. The assertion above it would have to fail for it to fail.